cellrank/tl/kernels/_base_kernel.py

```diff
@@ -223,6 +227,85 @@ def kernels(self) -> List["Kernel"]:
         """Get the kernels of the kernel expression, except for constants."""
         return list(set(self._get_kernels()))
 
+    def compute_projection(
+        self,
+        basis: str = "umap",
+        key_added: Optional[str] = None,
+        copy: bool = False,
+    ) -> Optional[np.ndarray]:
+        """
+        Compute a projection of the transition matrix in the embedding.
+
+        The projected matrix can be then visualized as::
+
+            scvelo.pl.velocity_embedding(adata, vkey='T_fwd', basis='umap')
+
+        Parameters
+        ----------
+        basis
+            Basis for which to compute the projection.
```

Comment:
Basis for which to compute the projection, accessed via :paramref:`adata` ``.obsm['X_{basis}']``